app/views/admin/members/index.html.haml

```diff
@@ -52,7 +52,7 @@
               %td= link_to "#{member.infix}", member
               %td= link_to "#{member.last_name}", member
               %td
-                = link_to "tel:#{member.phone_number}" do
+                = link_to "tel:#{TelephoneNumber.parse(member.phone_number).international_number}" do
```

Comment:
If we enforce that every phone number in the database is in the international format, do we then also need to parse all the numbers again when showing them?

Reply:
I'm removing this because it doesn't change the formatting in any way.

It's not required to parse the numbers when showing them, the format in the database is still readable.
On the member show page it does change the formatting though (+31 6 1234567890 instead of +3161234567890) so that's the reason I parsed it again there. Alternatively we could just skip the parsing and display the number without any formatting.